jsx/Breadcrumbs.js

```diff
@@ -14,25 +16,104 @@
  */
 class Breadcrumbs extends React.Component {
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayCount: 0
+    };
+
+    this.checkScreenSize = this.checkScreenSize.bind(this);
+  }
+
+  componentWillMount() {
+    this.checkScreenSize();
+    if (typeof window !== 'undefined')
+      window.addEventListener('resize', this.checkScreenSize);
+  }
+
+  componentWillUnmount() {
+    if (typeof window !== 'undefined')
+      window.removeEventListener('resize', this.checkScreenSize);
+  }
+
+  checkScreenSize() {
+    // Used to check to current window size and
+    // sets the number of breadcrumbs to show
+    var windowWidth = window.innerWidth;
+    var displayCount = 4;
+
+    if (windowWidth <= 500) {
```

Comment:
These widths are loosely calculated using the min-widths set by the css for the given widths. They are somewhat optimized so that the breadcrumbs don't split into two rows